storybook/stories/API/props/EventHandlers.ts

```diff
@@ -0,0 +1,340 @@
+/*
+ * This file both exports the documentation of shared props separately, to be reused in places where only single props
+ * are documented, as well as grouped in case a whole group is needed.
+ */
+
+export const onAbort = { table: { category: 'EventHandlers' } };
+export const onAbortCapture = { table: { category: 'EventHandlers' } };
+export const onAnimationEnd = {
+  table: {
+    category: 'EventHandlers',
+    description: 'The customized event handler of animation end',
+  },
+};
+export const onAnimationEndCapture = { table: { category: 'EventHandlers' } };
+export const onAnimationIteration = { table: { category: 'EventHandlers' } };
+export const onAnimationIterationCapture = { table: { category: 'EventHandlers' } };
+export const onAnimationStart = {
+  table: {
+    description: 'The customized event handler of animation start',
+    category: 'EventHandlers',
+  },
+};
+export const onAnimationStartCapture = { table: { category: 'EventHandlers' } };
```

Comment:
idk if everyone of these need exported individually but eh, looks good

Reply:
Probably not, but am not certain whether all components support all EventHandler? 🤔